api/auth_test.go

```diff
@@ -122,10 +155,63 @@ func TestAuth_MFALoginTwoPhase(t *testing.T) {
 				return
 			}
 
+			if tt.wantErr {
+				if secret != nil {
+					t.Errorf("MFALogin() returned non-nil secret when error was expected")
```

Comment:
This new test is failing as the assertion is incorrect, on errors a secret is also returned with a warning set.